web_server/social_distribution/utils/endpoint_utils.py

```diff
@@ -169,11 +169,11 @@ def handle(self, request):
         :param request:
         :return:
         """
-        if self.requires_authentication and not request.user.is_authenticated:
-            return JsonResponse({
-                "success": False,
-                "message": "You must be logged in to access this Endpoint"
-            }, status=403)
+        #if self.requires_authentication and not request.user.is_authenticated:
+         #   return JsonResponse({
+          #      "success": False,
+           #     "message": "You must be logged in to access this Endpoint"
+            #}, status=403)
```

Comment:
Why are you disabling login checks?

Reply:
I was testing and I forgot to remove the comments